subscription-manager.spec

```diff
@@ -875,6 +880,9 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 # syspurpose
 %dir %{python_sitearch}/syspurpose
 %{python_sitearch}/syspurpose/*.py
+%if 0%{?suse_version}
+%{python_sitearch}/syspurpose/*.pyc
+%endif
```

Comment:
Most likely we can avoid this SUSE-only block by turning the `*.py` pattern in the line before to `*.py*`, in a similar way to what is done few lines above for rhsmlib.